docs/literate/how_to/hide_deco.jl

```diff
@@ -54,14 +56,18 @@ plot_erpimage!(
     dat;
     layout = (; hidespines = (), hidedecorations = (), use_colorbar = false),
 )
+
+
+# # Showing
+
 #=
 Some plots hide features by default. This could be reverted by setting the variables to `nothing`
 =#
 
 data, positions = TopoPlots.example_data()
 plot_topoplot(
-    data[:, 340, 1];
-    positions = positions,
+    data[:, 340, 1],
+    positions;
```

Comment:
this change depends on our discussion in #227. As indicated there, I'm not convinced this is a useful breaking change